NitroxClient/GameLogic/PlayerGameObjectManager.cs

```diff
@@ -17,6 +20,23 @@ public void UpdatePlayerPosition(string playerId, Vector3 position, Quaternion r
             MovementHelper.MoveGameObject(player, position, rotation, PLAYER_TRANSFORM_SMOOTH_PERIOD);
         }
 
+        public void UpdateAnimation(string playerId, AnimChangeType type, AnimChangeState state)
+        {
+            GameObject player = GetPlayerGameObject(playerId);
+            GameObject playerView = player.transform.Find("player_view").gameObject;
+            AnimationController controller = playerView.GetComponent<AnimationController>();
+
+            bool animationValue;
+
+            switch (type)
+            {
+                case AnimChangeType.Underwater:
+                    animationValue = (state != AnimChangeState.Off);
```

Comment:
should this be on the same line as animationValue's initialization?  bool animationValue = (state != AnimChangeState.Off)